src/Integration/ModulesIntegration.php

```diff
@@ -27,7 +27,7 @@ final class ModulesIntegration implements IntegrationInterface
     public function setupOnce(): void
     {
         Scope::addGlobalEventProcessor(function (Event $event) {
-            $self = Hub::getCurrent()->getIntegration(self::class);
+            $self = SentrySdk::getIntegration(self::class);
 
             if ($self instanceof self) {
                 self::applyToEvent($self, $event);
```

Comment:
I know it's not really the scope here, but why bother with getting the `$self` from the sdk + checking, and not use `$this`... directly ? same in the `applyToEvent` method, which would transform itself from static to non-static

(Same for the `RequestIntegration`, I was about to make a PR for that...)

Reply:
With this, we can then remove the hard dependency on the sdk, and thus for monolog integration etc, we can be free of the hub / sdk / whatever and just apply whatever integration we want.

Reply:
Explanation about why this is needed has been given in #868